src/move_basic.cpp

```diff
@@ -199,7 +204,7 @@ static void getPose(const tf2::Transform& tf, double& x, double& y, double& yaw)
 
 // Constructor
 
-MoveBasic::MoveBasic(): tfBuffer(ros::Duration(30.0)),
+MoveBasic::MoveBasic(): tfBuffer(ros::Duration(3.0)),
```

Comment:
What is the rationale for the smaller buffer?

Reply:
To not have old observations be taken into account.